experimental/dds/tree2/src/core/forest/forest.ts

```diff
@@ -99,7 +100,16 @@ export interface IForestSubscription extends Dependee, ISubscribable<ForestEvent
 	 * This dummy node can be used to read the detached fields,
 	 * but other operations (such as inspecting the dummy node's type or path) should not be relied upon.
 	 */
-	moveCursorToPath(destination: UpPath | undefined, cursorToMove: ITreeSubscriptionCursor): void;
+	moveCursorToPath(destination: UpPath, cursorToMove: ITreeSubscriptionCursor): void;
+
+	/**
+	 * The cursor is moved to a special dummy node above the detached fields.
+	 * This dummy node can be used to read the detached fields,
+	 * but other operations (such as inspecting the dummy node's type or path) should not be relied upon.
+	 * As this method does not return a forest cursor and cannot be freed, the caller of this method must
+	 * use with caution and ensure that it is not used incorrectly.
+	 */
+	getRootCursor(): ITreeCursorSynchronous;
```

Comment:
People might expect this to point to the document root. TO help emphasize that it does not, I think naming it "getCursorAboveDetachedFields" would help.

Reply:
renamed to `getCursorAboveDetachedFields`